plugin/evm/validators/state_test.go

```diff
@@ -216,71 +217,34 @@ func TestStateListener(t *testing.T) {
 	db := memdb.New()
 	state, err := NewState(db)
 	require.NoError(err)
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
```

Comment:
nit: I think calls to Finish are not required in recent gomock versions.

Reply:
let's keep it for now